tests/SAML2/XML/saml/AuthenticatingAuthorityTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SAML2\XML\saml;
+
+use SAML2\Constants;
+use SAML2\DOMDocumentFactory;
+use SAML2\Utils;
+
+/**
+ * Class \SAML2\XML\saml\AuthenticatingAuthorityTest
+ */
+class AuthenticatingAuthorityTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @return void
+     */
+    public function testMarshalling(): void
+    {
+        $authority = new AuthenticatingAuthority('https://sp.example.com/SAML2');
+
+        $this->assertEquals(
+            strval($authority),
+            '<saml:AuthenticatingAuthority xmlns:saml="' . Constants::NS_SAML
+                . '">https://sp.example.com/SAML2</saml:AuthenticatingAuthority>'
+        );
```

Comment:
A minor detail here: the parameters to `assertEquals()` are the expected value and the actual value, in that order, so we should probably invert them here to avoid confusions in case of an error.